index.html

```diff
@@ -60,13 +60,164 @@ <h3>I can help you build a product , feature or website Look through some of my
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+          <button class="btn2">See Project </button> </form>
+
+          <form action="#" class="project2" id="project22">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+          <button class="btn2">See Project </button> </form>
+
+          <form action="#" class="project2" id="project21">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+          <button class="btn2">See Project </button> </form>
+
+
+          <form action="#" class="project2" id="project20">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
 
+          <button class="btn2">See Project </button> </form>
+        
+                     
+    </div>
+           
+
+
+
+       <div class="body3">
+        <h1 class="s1h11">About me</h1>
+
+        <h3>Hello I’m a software developer! I can help you build a product 
+            , feature or website Look through some of my work and experience! If you like what you see and have
+             a project you need coded, don’t hestiate to contact me.</h3>
+    
+
+             <form class="form2" action="https://www.linkedin.com/in/abdelruhman-mihamed-a42667179/overlay/1635484731651/single-media-viewer/"><button class="btn11">Get My Resume </button> </form> 
```

Comment:
- [ ] I recommend using `<a>` tag for links instead of `<form>` tag. Remember that we use the `<form>` tag only for forms.

Reply:
I cant use <a> tags because the linters read it as an issue "line 102  col 24   Error  The element “button” must not appear as a descendant of the “a” element
"